test/index.dart

```diff
@@ -0,0 +1,9 @@
+import 'box_test.dart' as box_test;
+import 'query_test.dart' as query_test;
+import 'query_property_test.dart' as query_property_test;
+
+void main() {
+  box_test.main();
+  query_test.main();
+  query_property_test.main();
+}
```

Comment:
@vaind Can this file be removed? Not sure why this was added by @Buggaboo.

Reply:
Yeah, definitely. It wasn't added lately - it's a leftover from the old state of main/dev (you can see that in the file's history)